_includes/v22.1/misc/table-storage-parameters.md

```diff
@@ -1,6 +1,6 @@
 | Parameter name      | Description | Data type | Default value |
 |---------------------+----------------------+-----+------|
-| `exclude_data_from_backup` | Excludes the data in this table from any future backups. | Boolean | `false` |
+| **New in v22.1:** `exclude_data_from_backup` | Excludes the data in this table from any future backups. | Boolean | `false` |
```

Comment:
```suggestion
| `exclude_data_from_backup` | **New in v22.1:** Excludes the data in this table from any future backups. | Boolean | `false` |
```

Reply:
I've been putting them in the description. Otherwise the table format looks bad.

Reply:
Ah, makes sense.